docker-compose.yml

```diff
@@ -1,17 +1,18 @@
-version: '3'
+version: '2'
 
 services:
   db:
-    image: postgres
+    image: postgres:alpine
     volumes:
-      - ./sql_init.sh:/docker-entrypoint-initdb.d/init_user_db.sh
-      - ./postgres-data:/var/lib/postgresql/data
+      - './sql_init.sh:/docker-entrypoint-initdb.d/init_user_db.sh:ro'
+      - 'pg_data:/var/lib/postgresql/data'
+
   web:
     build: .
-    command: python -m flask run --host 0.0.0.0
-    volumes:
-      - .:/contextualise
     ports:
-      - "5000:5000"
+      - '5000:5000'
```

Comment:
This isn't an improvement either.

Reply:
This wasn't designed to be an improvement, it just happened this way after the rewrite. However I would argue it has value since double qoutes can do variable interpolation while single cannot but I have changed it back.